GNUmakefile.osx

```diff
@@ -102,6 +102,7 @@ endif
 
 ifndef LIBXML2_LIBS
     LIBXML2_LIBS := $(shell xml2-config --libs)
+    LIBXML2_LIBS := $(filter-out -L$(shell xcrun --show-sdk-path)/usr/lib, $(LIBXML2_LIBS))
```

Comment:
Well anyway, you've already done the lion's share of the work, I'll just take the commit and add the comments myself.  Thanks.